### ISSUE TITLE
Unexpected aggressive-promise-polyfill error on dbwtester

### ISSUE THREAD

**Issue by paulirish** [GoogleChrome/lighthouse#11803]
This has come up in [some](https://github.com/GoogleChrome/lighthouse/pull/11799/checks?check_run_id=1526977435) smoketests recently......
![image](https://user-images.githubusercontent.com/39191/101700896-b4248480-3a32-11eb-882e-fdab79292629.png)

It's entirely clear where it's from.  Seems to only happen on ToT.

Me and @brendankenny can repro locally:

![image](https://user-images.githubusercontent.com/39191/101700852-a0791e00-3a32-11eb-9366-0398fdbd7d4b.png)

My repro:

* get chromium ToT
* open http://localhost:10200/dobetterweb/dbw_tester.html in it
* use 'pause on caught exceptions' carefully to find the cascade of exceptions that leads to `TypeError: the given value is not a Promise`

there's a lot of unknowns here..

---

I'm going to bypass the error temporarily in https://github.com/GoogleChrome/lighthouse/pull/11799

**Comment by brendankenny** [comment]
This is my minimal version of `dbw_tester.html` that repros. All parts seem necessary (though refresh+cache might be fooling me on some of it):

```html
<!doctype html>

<html>

<!-- If just this removed (or moved after jquery), will `Aw, Snap!` Chrome 89.0.4351.0. Removing any attributes does the same, too. -->
<link rel="preload" href="./dbw_tester.css" as="style" onload="this.rel = 'stylesheet'">

<!-- Where error occurs. Order of the scripts doesn't seem to matter. -->
<script src="./third_party/aggressive-promise-polyfill.js"></script>

<!-- If removed, no error. -->
<script type="module" src="./empty_module.js"></script>

<!-- If removed, no error. -->
<script src="http://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>

</html>
```

**Comment by brendankenny** [comment]
One notable thing in ToT Chromium today is top-level await (https://chromium.googlesource.com/chromium/src/+/19be7d71a9b41322599f02f0e9e0a7acbbff5943). I wonder if that could be affecting Promise stuff.

**Comment by connorjclark** [comment]
Maybe we should be tracking Chrome Dev instead of ToT.

**Comment by paulirish** [comment]
This error seems to also turn into a crash. 

When there's a crash there's a 'Inspector.targetCrashet` event, but it's logged in verbose.
After that, LH tries "Page appears to be hung, killing JavaScript..."  and soon after we see a `PROTOCOL_TIMEOUT` w/ `Emulation.setScriptExecutionDisabled`.  (what happened in the w/o JS removal PR: https://github.com/GoogleChrome/lighthouse/runs/1534292951 )

---------

I bisected the crash on chromium....


```sh
python2.7 bisect_builds.py --use-local-cache -o -a mac -g 89.0.4343.0 -b 89.0.4351.0 --  --disable-features=TranslateUI --disable-extensions --disable-component-extensions-with-background-pages --disable-background-networking --disable-sync --metrics-recording-only --disable-default-apps --mute-audio --no-default-browser-check --no-first-run --disable-backgrounding-occluded-windows --disable-renderer-backgrounding --disable-background-timer-throttling --force-fieldtrials="*BackgroundTracing/default/" --remote-debugging-port=55027 --user-data-dir=/var/folders/py/rf3cszrn5kdbj_2w2jd_8wsc003dq2/T/lighthouse.XXXXXXX.qOGcKFSb --enable-logging --v=0
```

```sh
node lighthouse-cli/index.js http://localhost:10200/dobetterweb/dbw_tester.html -port=55027 --config-path=blahblahah
```


> You are probably looking for a change made after 835252 (known good), but no later than 835253 (first known bad).
> CHANGELOG URL:
> The script might not always return single CL as suspect as some perf builds might get missing due to failure.
>   https://chromium.googlesource.com/chromium/src/+log/56d85fa5f642954381267744e440faaa74814479..3904e0fe3b6a83c4aa22c960a28558612178870f


.... @brendankenny called it. 😁

**Comment by paulirish** [comment]
Filed upstream chromium bug: https://bugs.chromium.org/p/chromium/issues/detail?id=1157692

**Comment by paulirish** [comment]
A fix has landed upstream: https://bugs.chromium.org/p/chromium/issues/detail?id=1157692#c4

**Comment by paulirish** [comment]
fixed by #11836
